scripts/get_counts.py

```diff
@@ -433,6 +445,17 @@ def MergeDevData(dest_count_dir, ngram_order):
 num_train_sets = int(f.readline())
 f.close()
 
+# set the memory restriction for "sort"
+sort_mem_opt = ("--buffer-size={0} ".format(args.max_memory))
+if (args.max_memory != '' and args.dump_counts_parallel == 'true'):
+  sub_memory = ParseInt(args.max_memory)/num_train_sets
```

Comment:
I considered this case. Since sort doesn't accept memory size such as 0.5G, if the result of division is zero, the code will exit and reminds user to provide a larger value as I wrote in lines 453-455. But I feel it's not a good solution. Any comment?